programming-extensions/programming-extension-processbuilder/src/test/java/org/objectweb/proactive/extensions/processbuilder/LinuxProcessBuilderTest.java

```diff
@@ -110,6 +110,32 @@ public void su_destroy_process() throws Exception {
         assertEquals(0, exitCode);
     }
 
+    @Test(timeout = 2000)
+    public void su_destroy_process_gracefully() throws Exception {
+        LinuxProcessBuilder processBuilder = new LinuxProcessBuilder(new OSUser(username, password), null,
+                proactiveHome);
+        Process process = processBuilder
+                .command("trap \"exit 5\" SIGTERM; echo $$; while true; do sleep 1; done")
+                .start();
+        process.destroy();
+        int exitCode = process.waitFor();
+        System.out.println("[SU_DESTROYPROCESS] process terminated with exit code " + exitCode);
+        assertEquals(5, exitCode);
+    }
+
+    @Test(timeout = 15000)
+    public void su_destroy_process_sigkill_after_graceful_timeout() throws Exception {
+        LinuxProcessBuilder processBuilder = new LinuxProcessBuilder(new OSUser(username, password), null,
+                proactiveHome);
```

Comment:
shouldn't you set proactive.process.builder.cleanup.time.seconds=2 to avoid waiting for a long period in the test ?